src/react-utils/useAutoFocus.ts

```diff
@@ -2,15 +2,20 @@ import {RefObject, useEffect} from 'react';
 
 export const useAutoFocus = (nodeRef: RefObject<HTMLElement>) => {
     useEffect(() => {
+        if (!nodeRef) {
+            return;
+        }
+
         const {current: anchor} = nodeRef;
         const timeout = setTimeout(() => {
             anchor?.focus();
         });
 
+        // eslint-disable-next-line consistent-return
```

Comment:
I suggest you return it as it was

https://react.dev/reference/rules/rules-of-hooks